tests/sentry/api/endpoints/test_account_identity_details.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import pytest
+
+from django.core.urlresolvers import reverse
+
+from social_auth.models import UserSocialAuth
+from sentry.testutils import APITestCase
+
+
+class AccountIdentityDetailsEndpointTest(APITestCase):
+    #  Throws backend not found
```

Comment:
Not sure why it's throwing :(

Reply:
is the social auth provider registered here? not sure if i'm asking the right question